modules/microshift-ops-config-embed-ostree.adoc

```diff
@@ -0,0 +1,162 @@
++
+[NOTE]
+====
+The default namespace security for `openshift-marketplace` is `baseline`, therefore a catalog source custom resource (CR) created in that namespace does not require a `spec.grpcPodConfig.securityContextConfig` value to be set. You can set a `legacy` or `restricted` value if required for the namespace and Operators you want to use.
+====
+
+. Get the SHA of the catalog index image by running the following command:
++
+[source,terminal]
+----
+$ skopeo inspect docker://registry.example.com/redhat/redhat-operator-index:v4.15 | jq `.Digest` <1>
+----
+<1> Pass the value from the `spec.image` field in the Catalog Source CR.
++
+.Example output
+[source,terminal]
+----
+"sha256:7a76c0880a839035eb6e896d54ebd63668bb37b82040692141ba39ab4c539bc6"
+----
+
+[IMPORTANT]
+====
+You must use the SHA instead of a tag in your catalog CR or the pod fails to start.
+====
+
+. Add the SHA of the catalog index commit to the Catalog Source (CR) by editing the YAML:
++
+.Example namespace `spec.image` configuration
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: cs-redhat-operator-index
+  namespace: openshift-marketplace
+spec:
+  image: registry.example.com/redhat/redhat-operator-index@sha256:7a76c0880a839035eb6e896d54ebd63668bb37b82040692141ba39ab4c539bc6 <1>
+  sourceType: grpc
+  displayName:
+  publisher:
+  updateStrategy:
+    registryPoll:
+      interval: 60m
+----
+<1> The SHA of the image commit.
+
+. Apply the YAML file from the results directory to the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f ./oc-mirror-workspace/results-1708508014/catalogSource-cs-redhat-operator-index.yaml
+----
++
+.Example output
+[source,terminal]
+----
+catalogsource.operators.coreos.com/cs-redhat-operator-index created
+----
+
+.Verification
+
+. Verify that the `CatalogSource` resources were successfully installed by running the following command:
++
+[source,terminal]
+----
+$ oc get catalogsource --all-namespaces
+----
+//add example output
+
+. Verify that the catalog source is running by using the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-marketplace
+----
++
+.Example output
+[source,terminal]
+----
+NAME                             READY   STATUS    RESTARTS   AGE
+cs-redhat-operator-index-4227b   2/2     Running   0          2m5s
+----
+
+. Create a Subscription CR configuration file by using the following example YAML:
++
+[source,terminal]
+----
+$ cat > <my-subscription-cr.yaml> <1>
+----
+<1> Replace _<my-subscription-cr.yaml>_ with the name of your Subscription CR.
++
+.Subscription custom resource example YAML
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: amq-broker
+  namespace: openshift-operators
+spec:
+  channel: 7.11.x
+  name: amq-broker-rhel8
+  source: cs-redhat-operator-index
+  sourceNamespace: openshift-marketplace
+----
```

Comment:
Same as before, the `cat` command isn't necessary. Users can create the file using `cat`, a text editor, or even the console.
```suggestion
. Create a `Subscription` CR, similar to the following example:
+
.Example `my-subscription-cr.yaml` file
[source,yaml]
----
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: amq-broker
  namespace: openshift-operators
spec:
  channel: 7.11.x
  name: amq-broker-rhel8
  source: cs-redhat-operator-index
  sourceNamespace: openshift-marketplace
----
```

Reply:
No console in MicroShift, but point taken.